packages/formstr-app/src/containers/CreateFormNew/components/AnswerSettings/index.tsx

```diff
@@ -93,7 +93,42 @@ function AnswerSettings() {
           />
         </div>
       </div>
+      <div
```

Comment:
Should this be in the question card next to choices ( like how it is for the question label) instead of in answer settings?